tests/test_dpo_trainer.py

```diff
@@ -944,18 +944,19 @@ def test_dpo_trainer_torch_dtype(self):
                 report_to="none",
             )
 
-            with pytest.raises(
-                ValueError,
-                match="Invalid `torch_dtype` passed to the DPOConfig. Expected a string with either `torch.dtype` or 'auto', but got -1.",
-            ):
+            with self.assertRaises(ValueError) as context:
                 _ = DPOTrainer(
                     model=self.model_id,
-                    ref_model=self.model_id,
```

Comment:
Why do you remove this?

Reply:
I apologize for mistakenly removing the ref_model parameter. This was an oversight on my part and I have now restored it in the latest commit.